core/embed/rust/src/lib.rs

```diff
@@ -51,17 +42,13 @@ fn panic_debug(panic_info: &core::panic::PanicInfo) -> ! {
 #[panic_handler]
 /// Default panic handling. Not showing any details - thus saving flash space.
 fn panic(_info: &core::panic::PanicInfo) -> ! {
-    use cstr_core::CStr;
-
-    // Although it would be ideal to use the original error message, ignoring it
-    // lets us avoid the `fmt` machinery and its code size and is also important for
-    // security reasons, as we do not always controls the message contents. We
-    // should also avoid printing "panic" or "rust" on the user screen to avoid any
-    // confusion.
-
-    // SAFETY: Safe because we are passing in \0-terminated literals.
-    let empty = unsafe { CStr::from_bytes_with_nul_unchecked(b"\0") };
-    let msg = unsafe { CStr::from_bytes_with_nul_unchecked(b"rs\0") };
-
-    trezorhal::common::fatal_error(empty, msg, empty, 0, empty);
+    // TODO: as of 2022, ignoring the `_info` parameter does not help with saving
```

Comment:
maybe state compiler version instead/in addition to year

Reply:
fixed in 863504278